phidgets_analog_outputs/src/analog_outputs_ros_i.cpp

```diff
@@ -0,0 +1,86 @@
+#include <memory>
```

Comment:
This file should have a copyright notice at the top (the fact that it is missing from https://github.com/ros-drivers/phidgets_drivers/blob/noetic/phidgets_digital_outputs/src/digital_outputs_ros_i.cpp is a bug).